### ISSUE TITLE
Light level not working if there is a Hi-tech door in area

### ISSUE THREAD

**Issue by ExDomino** [OreCruncher/DynamicSurroundings#335]
Minecraftt: 1.12
Forge: 14.23.3.2669
DynamicSurroundings: 3.4.12
MalisisCore: 6.4.0
MalisisDoors: 7.3.0

Description: Light level from DynamicSurroundings is not working if there is a Hi-tech door from the mod MalisisDoor in a visible chunk (depending of visible chunk in settings). Placing only one door of this kind stops the light level from working and destroying the door makes light level working again.

http://img.mod-minecraft.net/Mods/Malisis-Doors-Mod-18.png

**Comment by OreCruncher** [comment]
Yep - discovered this playing revelations.  Not sure what Malisis doors is doing.

**Comment by OreCruncher** [comment]
@Ordinastie  Do you have any idea of what may be going on?  I don't have a problem making tweaks to DS if it makes things better.  Relevant rendering code for DS light level can be found [here](https://github.com/OreCruncher/DynamicSurroundings/blob/master/src/main/java/org/blockartistry/DynSurround/client/hud/LightLevelHUD.java#L289).
